src/inc/msquic.h

```diff
@@ -1809,6 +1844,8 @@ typedef struct QUIC_API_TABLE {
     QUIC_CONNECTION_OPEN_IN_PARTITION_FN
                                         ConnectionOpenInPartition;   // Available from v2.5
 
+    QUIC_REGISTRATION_CLOSE_ASYNC_FN   RegistrationCloseAsync;       // Available from v2.5
```

Comment:
Let's move this to the end of the struct, include it in the preview functions, and mark it for v2.6 instead,